src/zmq/zmqpublishnotifier.cpp

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "zmqpublishnotifier.h"
+#include "main.h"
+#include "net.h"
```

Comment:
nit: I think this include can be removed. Compiles fine on OSX/Debian without this include.